### PR TITLE
Delete old data lookup APIs and adds datalist client to replace them

### PR DESCRIPTION
**Pull request by usbsnowcrash**
**Feature Branch**
This PR looks big but its mostly deletes

### CODE REVIEW

File: lib/cb/clients/data_list.rb [hunk 1]
```diff
@@ -8,15 +8,22 @@
...
-        Cb::Responses::Industry::Search.new(response)
+    class DataList < Base
+      class << self
+        # https://careerbuilder.readme.io/docs/consumerdatalistcountries
+        def get(args={})
+          uri = "#{ Cb.configuration.uri_data_list }/#{ args[:list] }"
+          cb_client.cb_get(uri, headers: headers(args), query: query_strings(args))
+        end
+
+        def query_strings(args)
```
Comment (oochelz) [inline lib/cb/clients/data_list.rb]: Should this be private?

File: lib/cb/config.rb [hunk 2]
```diff
@@ -61,30 +61,22 @@ def set_default_api_uris
...
-      @uri_desired_job_type ||= '/consumer/datalist/desiredjobtype'
-      @uri_education_code ||= '/v1/EducationCodes'
-      @uri_employee_types ||= '/v1/employeetypes'
+      @uri_data_list ||= '/consumer/datalist'
       @uri_job_branding ||= '/branding'
-      @uri_job_category_search ||= '/v1/categories'
       @uri_job_find ||= '/v3/Job'
-      @uri_job_industry_search ||= '/v1/industrycodes'
       @uri_job_insights ||= '/consumer/job-insights'
+      @uri_report_job ||= '/v1/job/report'
```
Comment (oochelz) [inline lib/cb/config.rb]: Why move this up here, out of alphabetical order?

File: spec/cb/clients/data_list_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2015 CareerBuilder, LLC
```
Comment (oochelz) [inline spec/cb/clients/data_list_spec.rb]: 2016

**Comment by oochelz** [comment]
Done reviewing. I glazed over the deletions, assuming you've got it all right because I can't really tell anyway.

**Comment by Casao** [comment]
So DataList client replaces them all? Cool, big change.

Makes sense to me -- if you want a merge tho update that changelog and version

**Comment by oochelz** [comment]
@Casao This is a feature branch PR. The main one into master will have changelog/version.

**Comment by oochelz** [comment]
But also... I have comments!

**Comment by Casao** [comment]
OH HEY IT IS IGNORE MEEEE

File: lib/cb/config.rb (hunk 2 above)
Comment (usbsnowcrash) [inline lib/cb/config.rb]: Copy pasta errors...fixing

**Comment by usbsnowcrash** [comment]
@oochelz @Casao Anything else?

**Pull request merged**